primitives/relayers/src/registration.rs

```diff
@@ -113,7 +128,7 @@ where
 
 	fn repatriate_reserved(
 		_relayer: &AccountId,
-		_beneficiary: RewardsAccountParams,
+		_beneficiary: ExplicitOrAccountParams<AccountId>,
```

Comment:
Maybe `ExplicitOrAccountParams` changes could go as a separate PR to `polkadot-sdk`, but not necessary